CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.2] - 2021-04-14
+
+### Changed
+
+* Dependencies updated for both frontend and backend
+* Improved httpGetJson to improve authentication and permit custom return types
+
+### Fixed
+
+* Fixed an issue where Ktor would cut a response if too long, causing crashes on the id access logs (profile page)
+* Fixed an issue where Logback would cause a stack overflow during a cyclic exception stack trace logging
+* Fixed an issue where httpGetJson calls by rulebooks would cause thread leaking (#262)
+* Increased login popup windows so that the Discord QR code can be displayed (#250)
+
```

Comment:
- Add proper links for all items. The links are not automatic, and you need to define them using regular Markdown (c.f. other updates)

- Also, you need to update the Unreleased link

- Moreover, you need to signal the deprecation of that thing in simple http whose name i forgot